core/src/main/java/io/aiven/kafka/tieredstorage/chunkmanager/cache/ChunkCacheConfig.java

```diff
@@ -33,6 +33,12 @@ public class ChunkCacheConfig extends AbstractConfig {
             + "where \"-1\" represents infinite retention";
     private static final long DEFAULT_CACHE_RETENTION_MS = 600_000;
 
+    private static final String CACHE_PREFETCHING_SIZE_CONFIG = "prefetching.bytes";
+    private static final String CACHE_PREFETCHING_SIZE_DOC =
+        "The amount of data that should be eagerly prefetched and cached";
+
+    private static final int CACHE_PREFETCHING_SIZE_DEFAULT = 0; //TODO find out what it should be
```

Comment:
Thinking about defaults, we can target something like 30MB.
The goal is to target 2x the max bytes per partition. If we suggest max fetch bytes per partition to be 15MB, using 30MB/s here will help use to have _the following_ fetch request prepared locally. 

More than 30MB may be too much in term of network allocation.